src/main/java/org/cristalise/kernel/lookup/ItemPath.java

```diff
@@ -37,6 +38,8 @@
 
     protected String mIOR;
 
+    protected boolean dependencyMemberPath;

```

Comment:
This XML fregment in the <Item> declaration has a misleading name. itemPath actually contains data for a DomainPath object:
`<DependencyMember itemPath="/desc/ActivityDesc/kernel/ManageScript">`